scripts/redact_cli_py/redact/utils/pdf_renderer.py

```diff
@@ -2,53 +2,25 @@
 # Licensed under the MIT License. See License.txt in the project
 # root for license information.
 
-import sys
 from PIL import Image
-
-import ctypes
-import pypdfium as pdfium
-
-WHITE = 0xFFFFFFFF
+import pypdfium2 as pdfium
 
 
 class PdfRenderer:
+
     def __init__(self):
-        # Initiate PDFium - This only needs to happen once
-        pdfium.FPDF_InitLibraryWithConfig(pdfium.FPDF_LIBRARY_CONFIG(2, None, None, 0))
+        pass
```

Comment:
@mara004 question: is it a good convention that we use `def __init__(self): pass` instead of removing it altogether?

Reply:
IDK, probably doesn't matter. Perhaps this wouldn't even need to be a class?